apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/rivers/OpprettJournalfoerOgDistribuerRiver.kt

```diff
@@ -61,23 +63,28 @@ class OpprettJournalfoerOgDistribuerRiver(
         packet: JsonMessage,
         context: MessageContext,
     ) {
-        val brevkode = packet[BREVMAL_RIVER_KEY].asText().let { Brevkoder.valueOf(it) }
-        // TODO: prøver å finne fornavn etternavn for Systembruker.brev altså "brev" else {
-        try {
-            runBlocking {
-                packet.brevId =
+        runBlocking {
+            val brevkode = packet[BREVMAL_RIVER_KEY].asText().let { Brevkoder.valueOf(it) }
+            // TODO: prøver å finne fornavn etternavn for Systembruker.brev altså "brev"
+
+            if (brevkode != Brevkoder.BP_INFORMASJON_DOEDSFALL) {
```

Comment:
Korfor denne if-en? Ser ut for meg som else-blokka er fin og kan brukast alltid?

Reply:
Jeg kjenner ikke flyten der helt. Tok den med siden den var der _før_ endringen du pushet nylig. Ser du fjernet den, men var ikke 100% sikker på at den burde fjernes 🤔

Reply:
Glem det. Gravde litt i kodehistorikken og da gav det plutselig mening hvorfor den hadde havnet der. Fjernet det nå 👍